doc/installation.rst

```diff
@@ -4,15 +4,11 @@ Installation
 Requirements
 ------------
 
-Installing the **garnett** package requires python 2.7.x or 3.x, and **numpy**.
+Installing the **garnett** package requires python 2.7.x or 3.x, **numpy** and **rowan**.
 
 With conda
 ----------
 
-.. sidebar:: Closed-source
-
-    **garnett** is not publicly released, installation via conda therefore requires access to the *private* Glotzer channel.
-
 To install the package with conda_, execute
```

Comment:
We'll need to revisit this when we actually prepare the conda package. At the very least, we'll need to change the install line to `conda install -c conda-forge garnett` and change the URL below to https://conda.io/. I would feel comfortable making those changes now, in preparation.

Reply:
pip will just become `pip install garnett` -- and we can drop the info about avoiding superusers.

Reply:
Done already, thanks.